### PR TITLE
release: 0.7.1

### PR DESCRIPTION
**Pull request by diegodelemos**

### CODE REVIEW

File: docs/openapi.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "info": {
     "description": "Submit workflows to be run on REANA Cloud",
     "title": "REANA Server",
-    "version": "0.7.0"
+    "version": "0.7.1"
```
Comment (tiborsimko) [inline docs/openapi.json:6]: Is the PR complete? Need to change also `CHANGES.rst`...

Comment (diegodelemos) [reply docs/openapi.json:6]: Updated.

File: CHANGES.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Changes
 =======
 
-Version 0.7.1 (UNRELEASED)
+Version 0.7.1 (2020-11-10)
 --------------------------
 
 - Fixes problem on GitLab integration preventing the synchronisation of projects with existing webhooks.
```
Comment (tiborsimko) [inline CHANGES.rst:7]: Can you please complete also WRT ping-related, restart-related, and perhaps also celery-related fixes?

```console
$ git log 0.7.0..upstream/pr/316 --format=oneline
f870cfa43f54d7b03581440341cf43754109d2c5 (upstream/pr/316) release: 0.7.1
fa209961becbd43fcb2a4b4dfaf11b98ffba7c5c (HEAD -> maint-0.7, upstream/pr/315, upstream/maint-0.7, origin/maint-0.7) gitlab: fix fetching projects with >
fc393cdf2d96b6bdf97c26cc5674033966478b83 (upstream/pr/312) rest: return reana_server url
32ab3e24ca46e623f98ece2322dacfd5d5a2b4fd (upstream/pr/294) rest: allow restart with different languages in same workspace
0d92dfb3490a4862ea05e5c50458d20dcb91550f (upstream/pr/310) setup: pin celery version <5
247e3a3adfcf298312fd7e85efc60aaf1bc2437d (upstream/pr/309) ci: added github actions workflow
```
```

File: CHANGES.rst [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
+- Fixes restarting of Yadage and CWL workflows.
+- Fixes Celery version less that 5 because of conflicting REANA-Commons' ``kombu`` dependency.
```
Comment (tiborsimko) [inline CHANGES.rst:9]: We don't really fix Celery here, so perhaps reformulate as:

- Fixes conflicting `kombu` installation requirements by requiring Celery version 4.

**Review by tiborsimko — APPROVED** [review]
LGTM, left one more comment WRT possibly shortening GitLab phrasing.

**Pull request merged**